File: leetcode/Algorithms/1876. Substrings of Size Three with Distinct Characters.cpp
class Solution {
public:
    int countGoodSubstrings(string s) {
        int count=0;
        if(s.size()<3) return 0;
        for(int i=0; i<s.length()-2; i++){
            char s1 = s[i];
            char s2 = s[i+1];
            char s3 = s[i+2];
            if(s1!=s2 && s1!=s3 && s2!=s3){
                count++;
            }
        }
        return count;
    }
};

File: leetcode/Algorithms/1876. Substrings of Size Three with Distinct Characters.py
class Solution:
    def countGoodSubstrings(self, s: str) -> int:
        count = 0
        for i in range(len(s)-2):
            s1 = s[i]
            s2 = s[i+1]
            s3 = s[i+2]
            if(s1!=s2 and s2!=s3 and s3!=s1):
                count+=1
            i+=1
        return count